docusaurus.config.js

```diff
@@ -6,7 +6,7 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: 'Apache Kvrocks (incubating)',
```

Comment:
`incubating` mark should be kept.

Reply:
`incubating` mark is not removed.
In the new UI, the styles of `Apache Kvrocks` and `incubating` are different. 
For the convenience of controlling the font style, I wrote the `incubating` in another place (src/pages/index.tsx).
You can see the preview provided by @ColinChamber  below, `incubating` is reserved.